cmd/config-migrate.go

```diff
@@ -2412,7 +2412,7 @@ func migrateV27ToV28() error {
 // Migrates '.minio.sys/config.json' v27 to v28.
 func migrateMinioSysConfig(objAPI ObjectLayer) error {
 	// Construct path to config.json for the given bucket.
-	configFile := path.Join(bucketConfigPrefix, minioConfigFile)
```

Comment:
was it not migrating properly before?

Reply:
we are I just changed it to the right value, it was a typo.